create/create.go

```diff
@@ -0,0 +1,139 @@
+package create
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	git "github.com/libgit2/git2go/v30"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/tarantool/cartridge-cli/project"
+	"github.com/tarantool/cartridge-cli/templates"
+)
+
+const (
+	initialTagName   = "0.1.0"
+	initialCommitMsg = "Initial commit"
+)
+
+func CreateProject(projectCtx project.ProjectCtx) error {
+	log.Infof("Creating an application %q...", projectCtx.Name)
+
+	// check that application doesn't exist
+	if _, err := os.Stat(projectCtx.Path); err == nil {
+		return fmt.Errorf("Application already exists in %s", projectCtx.Path)
+	}
+
+	var err error
+
+	err = os.Mkdir(projectCtx.Path, 0755)
```

Comment:
Above you use one-liner "if". Here you don't. It's a bit inconsistent. I think we should support uniformity where it's possible